### PR TITLE
docs: Update installation docs for pointing to Trivy Releases.

### PR DESCRIPTION
**Pull request by simar7**
Fixes: https://github.com/aquasecurity/trivy/issues/461

Signed-off-by: Simarpreet Singh <simar@linux.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,7 +120,7 @@ $ sudo yum -y install trivy
 or
 
 ```
-$ rpm -ivh https://github.com/aquasecurity/trivy/releases/download/v0.1.6/trivy_0.1.6_Linux-64bit.rpm
+$ rpm -ivh https://github.com/aquasecurity/trivy/releases/download/{TRIVY_VERSION}/trivy_0.1.6_Linux-64bit.rpm
```
Comment (mrueg) [inline README.md:123]: there's still 0.1.6 in the file name that would need to be replaced as well.

File: README.md [hunk 2]
```diff
@@ -137,7 +139,7 @@ or
 
 ```
 $ sudo apt-get install rpm
-$ wget https://github.com/aquasecurity/trivy/releases/download/v0.1.6/trivy_0.1.6_Linux-64bit.deb
+$ wget https://github.com/aquasecurity/trivy/releases/download/{TRIVY_VERSION}/trivy_0.1.6_Linux-64bit.deb
```
Comment (mrueg) [inline README.md:142]: Same here

File: README.md [hunk 3]
```diff
@@ -137,7 +139,7 @@ or
...
 $ sudo dpkg -i trivy_0.1.6_Linux-64bit.deb
```
Comment (mrueg) [inline README.md:143]: and here as well.

File: README.md (hunk 1 above)
Comment (simar7) [reply README.md]: Thanks @mrueg for catching that!

**Pull request merged**